File: mazal-tov/src/Classes/endMatching.ts
import { DecimalPipe } from '@angular/common';
export class endMatching{
     Id:number ;
     EndDate:Date ;
   UpdateDate:Date ;
  IsPaid:boolean ;
   WhoPaid :number;
   //לברר אם זה סוג טוב:
   Price:DecimalPipe; 
 Remark :string;
   IsCare:boolean ;
   EndPlace:string ;
 MatchingId:number ;
   

}

File: Mazal-Tov WebApi/DTO/endMatching.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class EndMatching
    {
        public int Id { get; set; }
        public DateTime EndDate { get; set; }
        public DateTime UpdateDate { get; set; }
        public bool IsPaid { get; set; }
        public int WhoPaid { get; set; }
        public decimal Price { get; set; }
        public string Remark { get; set; }
        public bool IsCare { get; set; }
        public string EndPlace { get; set; }
        public int MatchingId { get; set; }
       


    }
}
